pyproject.toml

```diff
@@ -1,69 +1,69 @@
 [build-system]
-requires = ["setuptools>=64", "setuptools_scm[toml]>=6.2", "wheel"]
+requires = ["setuptools>=64", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "epics-containers-cli"
+name = "edge-containers-cli"
 classifiers = [
     "Development Status :: 3 - Alpha",
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
 ]
-description = "One line description of your module"
-dependencies = ["natsort", "typer[all]", "requests", "ruamel.yaml", "jinja2"]
+description = "CLI for deploying and managing epics containers IOCs and services"
+
+dependencies = [
```

Comment:
Considering this is an application rather than a library, why are we not pinning dependencies?

Reply:
Sort of because I'm expecting the copier framework to make a constraints file for me. But it is not doing that yet - so you are correct we should pin the latest working set. I'd like to make sure dependabot is looking at these though. Otherwise they go stale quickly. Not sure if that is supported by dependabot or if the copier template uses it.

Reply:
Dependabot does support pyproject.yaml

Reply:
Kinda, but at the moment the behaviour is "blindly update to the latest version of everything, even if that combination is not pip installable". That means that once a week you get a PR where the CI will fail with "pip can't install this", which is useless spam. I've tracked the dependabot tickets in https://github.com/DiamondLightSource/python-copier-template/issues/36 but there doesn't seem to be any activity on their side...

Reply:
My instinct is to leave these unpinned for the moment and fix issues as they come up.
Actually pinning will likely break the test matrix and I don't want to spend ages finding the sweet spot of version restrictions that work across the matrix.

@coretl do you advocate?